api-reference/50 Common/Object Structures/positionConfig/at/y.md

```diff
@@ -5,6 +5,6 @@ type: String
 ---
 ---
 ##### shortDescription
-Holds the vertical position specifier.
+Specifies a position in the vertical direction.
```

Comment:
Can we say:
Specifies the vertical position.

Reply:
Can we change this to:
Specifies a position in the vertical direction (for top, bottom, or center alignment).